index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Playfair+Display&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section id="header">
+      
+        
+<div class="Menu">   
+<div class="Menu-line"></div>
+<div class="Menu-line"></div>
+<div class="Menu-line"></div>
+</div>   
+
+<img src="images/Shape (1).png" alt="" class="image1">
+<img src="images/Shape (2).png" alt="" class="image2">
```

Comment:
for every image tag in your project, let's add descriptive text for your `alt` tag as this helps screen readers understand what the image is about, also this will become useful when the image link is broken or it is taking time to load😉adding this will lead a better user experience😉